docs/METRICS.md

```diff
@@ -83,6 +85,15 @@ Event label: exactly what control invoked the action, such as toolbar-button.  T
 * `selection`: anything that happens during the selection phase, that doesn't happen in the topbar
 * `keyboard`: any keyboard shortcut used
 
+#### Login status custom dimension
+
+The `cd9` dimension holds the "login type". The values can be:
+
+* Empty: visitor is not logged in
```

Comment:
LGTM, as long as the dashboard allows filtering on empty values of custom dimensions

Reply:
Good catch! Updated to pass explicit value 'non-login' for not logged in case